model/Dataset/Classes/Dataset.md

```diff
@@ -68,3 +66,32 @@ Metadata information that can be added to a dataset that may be used in a softwa
   - type: DatasetAvailabilityType
   - minCount: 0
   - maxCount: 1
+
+## External properties restrictions
+
+- /Core/Element/name
+  - minCount: 1
```

Comment:
This is already set for `/Software/Package` and does not add a new restriction here. Can be removed.

Reply:
It helps make it clearer which properties, are mandatory for Dataset profile.   I don't see issue with leaving it hear for now until we have the auto generation of fields per profile working.

Reply:
It might not come at 0 cost and might introduce more complexity into generated documents like the ttl and the SHACL. And it makes it harder to find the actual changes.

Reply:
suggest the fix to remove it, and we'll revisit after the merge.